code/espurna/garland/color.h

```diff
@@ -0,0 +1,83 @@
+#if GARLAND_SUPPORT
+
+#ifndef color_h
+#define color_h
+#include <Arduino.h>
+
+struct Color
+{
+    union
+    {
+        struct
+        {
+            byte r;
+            byte g;
+            byte b;
+        };
+        byte raw[3];
+    };
+
+    inline Color() __attribute__((always_inline)) : r(0), g(0), b(0) {}
```

Comment:
It is, technically, already inline, even without the keyword. Below one as well, anything declared in the class body is inline implicitly